amm_info updates (#1704)
XRPLF/xrpl-dev-portal: content/references/http-websocket-apis/public-api-methods/path-and-order-book-methods/book_offers.md

The request includes the following parameters:

| `Field`        | Type             | Description                    |
|:---------------|:-----------------|:-------------------------------|
| `ledger_hash`  | [Hash][]         | _(Optional)_ A 20-byte hex string for the ledger version to use. (See [Specifying Ledgers][]) |
| `ledger_index` | [Ledger Index][] | _(Optional)_ The [ledger index][] of the ledger to use, or a shortcut string to choose a ledger automatically. (See [Specifying Ledgers][]) |

> | `limit`        | Number           | _(Optional)_ If provided, the server does not provide more than this many offers in the results. The total number of results returned may be fewer than the limit, because the server omits unfunded offers. |

Reviewer: ```suggestion
| `limit`        | Number           | _(Optional)_ The maximum number of offers to provide in the results. The server omits unfunded offers. |
```
